cdk/drupal/drupal_stack.py

```diff
@@ -25,19 +25,21 @@
 
 # When making a new development AMI:
 # 1) $ ave oe-patterns-dev make packer
-# 3) $ ave oe-patterns-dev make AMI_ID=ami-fromstep1 copy-image
+# 2) Update both AMI_ID and AMI_NAME immediately below
+AMI_ID="ami-06bfd9042474cab3f"
+AMI_NAME="ordinary-experts-patterns-drupal-0.1.0-86-g3c59022-20200527-0150"
+
+# 3) $ ave oe-patterns-dev make AMI_ID=ami-fromstep2 copy-image
 # 4) Copy the code that copy-image generates below
 
 # AMI list generated by:
-# make AMI_ID=ami-062b13bbef7bad514 copy-image
-# on Tue May 26 21:56:04 UTC 2020.
-AMI_ID="ami-062b13bbef7bad514"
-AMI_NAME="ordinary-experts-patterns-drupal-0.1.0-85-gdddc067-20200526-0934"
+# make AMI_ID=ami-06bfd9042474cab3f copy-image
+# on Wed May 27 13:59:02 UTC 2020.
```

Comment:
can you check you have the latest from develop and run the `copy-image` again and re-do the output part? I changed it.

Reply:
@dylanvaughn just updated the ami again with develop